addons/sale/i18n/ar.po

```diff
@@ -4111,7 +4111,7 @@ msgstr "تحليل المبيعات"
 
 #. module: sale
 #: model:ir.actions.act_window,name:sale.action_order_report_customers
-msgid "Sales Analysis By Customers"
+msgid "Sales Analysis Per Customers"
```

Comment:
`.po` files are handled by @C3POdoo and generated from Transifex, you should never update those manually.
Only the `.pot` file must be adapted.

Reply:
Oh didnt know, thanks!!